tasks/Task.cpp

```diff
@@ -112,12 +115,12 @@ base::samples::RigidBodyState Task::convertToRBS(const gps_ublox::PVT &data) con
         M_PI, Eigen::Vector3d::UnitX()) * may_invalidate(body2ned_velocity);
 
     auto geodetic = convertToBaseSolution(data);
-    base::samples::RigidBodyState nwu = mUTMConverter.convertToNWU(geodetic);
+    base::samples::RigidBodyState nwu = utmConverter.convertToNWU(geodetic);
     rbs.position = nwu.position;
     rbs.cov_position = nwu.cov_position;
     return rbs;
 }
-gps_base::SatelliteInfo Task::convertToBaseSatelliteInfo(const SatelliteInfo &sat_info) const
+static gps_base::SatelliteInfo convertToBaseSatelliteInfo(const SatelliteInfo &sat_info)
```

Comment:
Why it is needed to convert to `static`methods?

Reply:
> So, being a free function it has to be static.

"has to" is pretty strong here. It is better, sure, but it does not "have to".

Reply:
> "has to" is pretty strong here. It is better, sure, but it does not "have to".

Indeed. Sometimes it saves you from some linkage problems